system-addon/test/functional/mochitest/browser_highlights_section.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+registerCleanupFunction(async () => {
+  await clearHistoryAndBookmarks();
+});
+
+function refreshHighlightsFeed() {
+  // Toggling the pref will clear the feed cache and force a places query.
+  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", false);
+  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", true);
+}
+
+async function add_highlights_bookmark(count) {
```

Comment:
nit: the naming convention seems to be camcelCase as usual with test_names or things that create_tests with underscores.

Also, this could be generally useful as well from head.js